xLights/models/ArchesModel.h

```diff
@@ -23,6 +23,7 @@ class ArchesModel : public ModelWithScreenLocation<ThreePointScreenLocation>
         virtual void InitRenderBufferNodes(const std::string &type, const std::string &camera, const std::string &transform,
                                            std::vector<NodeBaseClassPtr> &Nodes, int &BufferWi, int &BufferHi, int stagger, bool deep = false) const override;
         virtual int GetNumPhysicalStrings() const override { return 1; }
+        virtual bool StrandsZigZagOnString() const override { return zigzag; }
```

Comment:
This function seems not used and probably has some cases that are wrong.  I would be happy to remove it if desired.